### PR TITLE
Fail if no loaded data can be found

### PR DESCRIPTION
**Pull request by jmelis**

### CODE REVIEW

**Comment by jmelis** [comment]
This PR depends on https://github.com/app-sre/qontract-server/pull/3

File: server.js [hunk 1]
```diff
@@ -39,11 +39,27 @@ const server = new ApolloServer({
...
+
+app.use(ensurePresentData);
+
 server.applyMiddleware({ app });
 
-app.get('/reload', (req, res) => { db.load(); res.send() });
-app.get('/health-check', (req, res) => { res.send() });
-app.get('/', (req, res) => { res.redirect('/graphql') });
+app.get('/reload', (req, res) => { db.load(); res.send(); });
+app.get('/health-check', (req, res) => {
```
Comment (jmelis) [inline server.js]: @riuvshin will this break the readinessProbe? The first few requests may fail until the data is loaded...

Comment (riuvshin) [reply server.js]: that should be fine as it will try few times and it has some delays.

File: models/db.js [hunk 2]
```diff
@@ -111,82 +100,135 @@ var labelFilter = function (label_filter, input_set) {
...
+      console.log(err, err.stack);
+    } else {
+      loadUnpack(data.Body.toString('utf-8'));
+    }
+  });
+};
 
-        db.datafile = dbDatafileNew;
-        db.datafiles = dbDatafilesNew;
+var loadFromFile = function () {
```
Comment (riuvshin) [inline models/db.js]: is this for running tests with local data?

Comment (jmelis) [reply models/db.js]: yes!

Comment (jmelis) [reply models/db.js]: supporting local data is dealt with in https://github.com/app-sre/qontract-server/pull/3

The purpose of this PR is to ensure that the server won't return 200 OK unless the DB was successfully loaded

Comment (riuvshin) [reply models/db.js]: oh very good!

**Pull request merged**